love/src/redux/selectors/selectors.js

```diff
@@ -536,6 +536,8 @@ export const getDomeState = (state) => {
     dropoutDoorState: domeData['event-ATDome-0-dropoutDoorState']?.[0].state?.value ?? 0,
     mainDoorState: domeData['event-ATDome-0-mainDoorState']?.[0].state?.value ?? 0,
     atDomeSummaryState: domeData['event-ATDome-0-summaryState']?.[0].summaryState?.value ?? 0,
+    //The following parameter is missing a CSC, add it when it becomes available//
```

Comment:
Please add a TODO at the beinning of the comment to easy access in the future:
```suggestion
    // TODO: The following parameter is missing a CSC, add it when it becomes available//
```